web/frontend/src/components/arm.vue

```diff
@@ -39,11 +39,24 @@ const fieldMap = [
   ['y', 'y'],
   ['z', 'z'],
   ['theta', 'theta'],
-  ['o_x', 'ox'],
-  ['o_y', 'oy'],
-  ['o_z', 'oz'],
+  ['o_x', 'oX'],
+  ['o_y', 'oY'],
+  ['o_z', 'oZ'],
 ] as const;
 
+const updateFieldMap: Record<string, Field> = {
+  // eslint-disable-next-line id-length
```

Comment:
I'd recommend just adding x, y, z to the list of exceptions in the eslint config since these are commonly used / acceptable single letter vars

Reply:
done https://github.com/viamrobotics/rdk/pull/2235/commits/e40b0f374ea99b8033c99838b2acbb95a213c5c2